### DISCUSSION TITLE
Copier on Podcast.__init__

### DISCUSSION THREAD

**Discussion by yajo** [copier-org/copier#329] [Show and tell]
Hi there community folks.

You might be interested in listening to [an interview made to myself about Copier, recently on Podcast.\_\_init\_\_](https://www.pythonpodcast.com/copier-project-scaffolding-episode-297/).

I hope I don't make you sleep. Regards.

**Comment by pawamoy** [comment]
Congrats! This is great 😄

**Reply by pawamoy** [reply]
Ha, you said my name perfectly :wink: :joy:

**Comment by pylang** [comment]
Copier was mentioned in today's episode of PythonBytes.

**Reply by yajo** [reply]
I'll listen to it, let's see what they say 😄 

FTR: https://pythonbytes.fm/episodes/show/219/htmx-dynamic-and-live-html-without-javascript and https://youtu.be/LAdksPSd6ZA?t=477

**Reply by yajo** [reply]
The youtube version helped me to see that we have to fix https://github.com/copier-org/copier/issues/323.

**Comment by pylang** [comment]
Yes indeed.  I was seeking the docs yesterday, but I didn't see them.  Today I realized there's a docs badge, but you may want to add an explicit link to the docs with two main points:

- How do I use this?  (with simple Example)
- How does this differ from cookiecutter?

Question: One thing I'm seeking is templating single files, which from what I recall, cookiecutter [does not handle](https://github.com/cookiecutter/cookiecutter/pull/885).   Can `copier` template single files?

**Reply by pawamoy** [reply]
On GitHub I got used to search the docs link in the top-right part of the repo main view :slightly_smiling_face: 
![Screenshot_2021-02-04_21-33-32](https://user-images.githubusercontent.com/3999221/106951980-c9ffae80-6730-11eb-94f6-f7bb21b99ef1.png)

**Reply by pylang** [reply]
Wow.  There is it.  My pardon.  I'm used to going straight to the README.  Haven't gotten used to GitHub's new look.  Thanks.

**Reply by pawamoy** [reply]
You're welcome :smile: That said, not all repos put this link here, so yeah, my second go-to is the badge, and then a link somewhere in the README ^^

**Reply by yajo** [reply]
> Can `copier` template single files?

I think this is a different subject that deserves its own new thread.
